samples/wifi/sr_coex/README.rst

```diff
@@ -0,0 +1,400 @@
+| Bluetooth LE only,     | N.A                | 1139               |
+| central                |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Bluetooth LE,| 10.2               | 1188               |
+| coexistence disabled   |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Bluetooth LE,| 10.2               | 1208               |
+| coexistence enabled    |                    |                    |
++------------------------+--------------------+--------------------+
+
+Shared antenna, Wi-Fi in 802.11n mode:
+
++------------------------+--------------------+--------------------+
+| Test case              | Wi-Fi UDP Tx       | Bluetooth LE       |
+|                        | throughput in Mbps | throughput in kbps |
++========================+====================+====================+
+| Wi-Fi only,            | 10.2               | N.A                |
+| client (UDP Tx)        |                    |                    |
++------------------------+--------------------+--------------------+
+| Bluetooth LE only,     | N.A                | 1180               |
+| central                |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Bluetooth LE,| 10.2               | 1177               |
+| coexistence disabled   |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Bluetooth LE,| 10.2               | 1191               |
+| coexistence enabled    |                    |                    |
++------------------------+--------------------+--------------------+
+
+Sample output
+=============
+
+The following screenshots show coexistence test results obtained for separate antenna configuration with Wi-Fi mode set to 802.11n.
+These tests were run with WLAN connected to an AP in 2.4 GHz band.
+In the images, left-side result shows Wi-Fi throughput that appears on a test PC terminal in which Wi-Fi **iperf** server is run and right-side result shows Bluetooth LE throughput that appears on a minicom terminal in which the Bluetooth LE throughput sample is run.
+
+.. figure:: images/wlan.jpg
+     :width: 711px
+     :align: center
+     :alt: Wi-Fi only throughput
+
+     Wi-Fi only throughput 10.2 Mbps
+
+.. figure:: images/ble.jpg
+     :width: 711px
+     :align: center
+     :alt: Bluetooth LE only throughput
+
+     Bluetooth LE only throughput: 1107 kbps
+
+.. figure:: images/wlan_ble_cd.jpg
+     :width: 711px
+     :align: center
+     :alt: Wi-Fi and Bluetooth LE CD
+
+     Wi-Fi and Bluetooth LE throughput, coexistence disabled: Wi-Fi 9.9 Mbps and Bluetooth LE 145 kbps
+
+.. figure:: images/wlan_ble_ce.jpg
+     :width: 711px
+     :align: center
+     :alt: Wi-Fi and Bluetooth LE CE
+
```

Comment:
@richabp, FYI the images might need rework (as discussed offline) and might need to be moved into the common images folder.